src/services/offline/MockModels.ts

```diff
@@ -147,3 +150,15 @@ export const OfflineAccountNames = (address: Address) => new AccountNames(addres
 export const OfflineNamespaceNames = (namespaceId: NamespaceId) => new NamespaceName(namespaceId, 'mocknamespace');
 
 export const OfflineMultisigAccountGraphInfo = new MultisigAccountGraphInfo(new Map());
+
+export const OfflineNetworkCurrencies = (networkType: NetworkType): NetworkCurrencies => {
+    const publicCurrency = new Currency({
```

Comment:
These values should be loaded from the NetworkConfig.ts instead of having a copy. 
Also, remember we would like to support private networks eventually.